website/cue/reference/components/sources/base/pulsar.cue

```diff
@@ -541,6 +541,35 @@ base: components: sources: pulsar: configuration: {
 		required:    false
 		type: string: examples: ["subscription_name"]
 	}
+	tls_options: {
+		description: "TLS options configuration for the Pulsar client."
+		required:    false
+		type: object: options: {
+			allow_insecure_connection: {
```

Comment:
I think this should be called `verify_certificate` in order to match the existing options.

Reply:
Thanks, fixed in [ed01093](https://github.com/vectordotdev/vector/pull/22148/commits/ed01093b8506a1dec3502446412208d395187ce0)